satellite/rewards/rewards_test.go

```diff
@@ -62,6 +62,8 @@ func TestOffer_Database(t *testing.T) {
 		for i := range validOffers {
 			new, err := planet.Satellites[0].DB.Rewards().Create(ctx, &validOffers[i])
 			require.NoError(t, err)
+			new.ExpiresAt = new.ExpiresAt.Round(time.Microsecond)
```

Comment:
why do we need to round here?

Reply:
We want to see if `new` exists in `all`. `all` is pulled from the database and the database rounds the timestamps to the nearest microsecond. Since the timestamps in `new` are not rounded, the `contains` check fails

Reply:
I thought postgresql truncated times to microseconds instead of rounding. Are you pretty sure it's rounding?

Reply:
Nope you're right it's rounding.